torchrl/envs/vec_env.py

```diff
@@ -39,6 +38,12 @@ def decorated_fun(self: _BatchedEnv, *args, **kwargs):
     return decorated_fun
 
 
+def _sort_keys(element):
+    if isinstance(element, tuple):
+        return "".join(element)
```

Comment:
let's use a separator that is unlikely to be used in the tensordict
(for instance we may have a key `"nextobs"` and a key `("next", "obs")`)
I sometimes use `"_-|-_"` or similar, take your pick :)

Reply:
Thanks, this is very useful toknow. Fixed